index.js

```diff
@@ -494,7 +507,8 @@ function flushSync (stream) {
     }
 
     if (++spins === 10) {
-      throw new Error('_flushSync took too long (10s)')
+      destroy(stream, new Error('_flushSync took too long (10s)'))
```

Comment:
I think all `Sync` methods should keep throwing. Those are often used in the shutdown phase of thread-stream and pino and we might no have an event loop at all.